src/vm/perfinfo.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PERFINFO_H
+#define PERFINFO_H
+
+
+#include "sstring.h"
+#include "fstream.h"
+
+class PerfInfo {
+public:
+    PerfInfo(int pid);
+    ~PerfInfo();
+    void LogImage(PEFile* pFile, WCHAR* guid); 
+
+private:
+    CFileStream* m_Stream;
+
+    const char sDelimiter = ';';
+
+    void OpenFile(SString& path);
+
+    void WriteLine(SString& type, SString& value);
```

Comment:
You should add a comment describing what this method does, and that it is for use within methods that log specific info (e.g. LogImageLoad).